Controller/Comment.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @author           Serge Kishiko <sergekishiko@gmail.com>
+ * @copyright        (c) 2015-2017, Pierre-Henry Soria. All Rights Reserved.
+ * @license          Lesser General Public License <http://www.gnu.org/copyleft/lesser.html>
+ * @link             http://hizup.uk
+ */
+
+namespace TestProject\Controller;
+
+class Comment
```

Comment:
Same thing here. Might be a good idea to extend to Blog, so you won't have to initialize session and you will be able to reuse the parent `$oUtil` object.